synapse/models/orgs.py

```diff
@@ -87,56 +143,27 @@ def getBaseModels():
                     ('seen:max', {'ptype': 'time:max'}),
                 )),
-                ('ou:hasfile', {}, [
-                    ('org', {'ptype': 'ou:org', 'ro': 1}),
-                    ('file', {'ptype': 'file:bytes', 'ro': 1}),
-                    ('seen:min', {'ptype': 'time:min'}),
-                    ('seen:max', {'ptype': 'time:max'}),
-                ]),
-                ('ou:hasfqdn', {}, [
-                    ('org', {'ptype': 'ou:org', 'ro': 1}),
-                    ('fqdn', {'ptype': 'inet:fqdn', 'ro': 1}),
-                    ('seen:min', {'ptype': 'time:min'}),
-                    ('seen:max', {'ptype': 'time:max'}),
-                ]),
-                ('ou:hasipv4', {}, [
-                    ('org', {'ptype': 'ou:org', 'ro': 1}),
-                    ('ipv4', {'ptype': 'inet:ipv4', 'ro': 1}),
-                    ('seen:min', {'ptype': 'time:min'}),
-                    ('seen:max', {'ptype': 'time:max'}),
-                ]),
-                ('ou:hashost', {}, [
-                    ('org', {'ptype': 'ou:org', 'ro': 1}),
-                    ('host', {'ptype': 'it:host', 'ro': 1}),
-                    ('seen:min', {'ptype': 'time:min'}),
-                    ('seen:max', {'ptype': 'time:max'}),
-                ]),
-                ('ou:hasemail', {}, [
-                    ('org', {'ptype': 'ou:org', 'ro': 1}),
-                    ('email', {'ptype': 'inet:email', 'ro': 1}),
-                    ('seen:min', {'ptype': 'time:min'}),
-                    ('seen:max', {'ptype': 'time:max'}),
-                ]),
-                ('ou:hasphone', {}, [
-                    ('org', {'ptype': 'ou:org', 'ro': 1}),
-                    ('phone', {'ptype': 'tel:phone', 'ro': 1}),
-                    ('seen:min', {'ptype': 'time:min'}),
-                    ('seen:max', {'ptype': 'time:max'}),
-                ]),
-                ('ou:haswebacct', {}, [
-                    ('org', {'ptype': 'ou:org', 'ro': 1}),
-                    ('web:acct', {'ptype': 'inet:web:acct', 'ro': 1}),
+                ('ou:org:has', {}, [
+                    ('org', {'ptype': 'ou:org', 'ro': 1, 'req': 1,
+                        'doc': 'FIXME The org who owns or controls the object or resource.'}),
+                    ('xref', {'ptype': 'propvalu', 'ro': 1, 'req': 1,
+                        'doc': 'FIXME The object or resource (prop=valu) that is owned or controlled by the org.'}),
+                    ('xref:node', {'ptype': 'ndef', 'ro': 1, 'req': 1,
+                        'doc': 'FIXME The ndef of the node that is owned or controlled by the org.'}),
+                    ('xref:prop', {'ptype': 'str', 'ro': 1,
+                        'doc': 'FIXME The property (form) of the object or resource that is owned or controlled by the org.'}),
                     ('seen:min', {'ptype': 'time:min'}),
                     ('seen:max', {'ptype': 'time:max'}),
                 ]),
+
```

Comment:
For seen:min / seen:max:
"The earliest known time when the org owned or controlled the resource."
"The most recent known time when the org owned or controlled the resource."

Reply:
fixed

Reply:
I made the same change to the ps:person:has and ps;persona:has docs since they had the same docs for the other props